planning_launch/config/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/stop_line.param.yaml

```diff
@@ -4,5 +4,6 @@
       stop_margin: 0.0
       stop_check_dist: 2.0
       stop_duration_sec: 1.0
+      use_initialization_stop_line_state: false
```

Comment:
I think it is ok to true by default

Reply:
@yukkysaito 
Before this PR's change, it was fixed to false.
But I changed it to ros_param so that it can be changed for each project.
So I think it should be false by default..

Reply:
@yukkysaito  @tkimura4 
I changed it to true.